app.go

```diff
@@ -43,7 +41,17 @@ func NewApp(db *sql.DB) (*App, error) {
 			return nil, err
 		}
 
+		acc, err := pksql.New(
+			db,
+			&projections.AccountProjectionHandler{},
+			nil,
+		)
+		if err != nil {
+			return nil, err
+		}
+
 		app.CustomerProjection = cust
+		app.AccountProjection = acc
```

Comment:
As the number of these projections grow, you might consider assigning to the struct fields directly, like:

```go
app.CustomerProjection, err  = pkgsql.New(...)
```

You'd likely need to pre-declare the `err` variable (`var err error`). This is just a suggestion, I'll leave it up to you if you want to do that.

Reply:
Yeah ok, that sounds good.

Reply:
Apparently Go won't let me do that?

Reply:
Danil helped me.